http-client/src/main/java/io/airlift/http/client/jetty/JettyClientDiagnostics.java

```diff
@@ -22,20 +22,20 @@
 import static java.util.Objects.requireNonNull;
 
 @ThreadSafe
-public class JettyClientDiagnostics
```

Comment:
Nothing in this commit seems to depend on this. Do this in the commit that adds this class instead.